lib/ansible/modules/network/ingate/ig_unit_information.py

```diff
@@ -0,0 +1,176 @@
+  ig_unit_information:
+    client:
+      version: v1
+      scheme: http
+      address: 192.168.1.1
+      username: alice
+      password: foobar
+'''
+
+RETURN = '''
+unit-information:
+  description: Information about the unit
+  returned: success
+  type: complex
+  contains:
+    installid:
+      description: The installation identifier
+      returned: success
+      type: string
+      sample: any
+    interfaces:
+      description: List of interface names
+      returned: success
+      type: string
+      sample: eth0 eth1 eth2 eth3 eth4 eth5
+    lang:
+      description: The unit's language
+      returned: success
+      type: string
+      sample: en
+    lic_email:
+      description: License email information
+      returned: success
+      type: string
+      sample: example@example.com
+    lic_mac:
+      description: License MAC information
+      returned: success
+      type: string
+      sample: any
+    lic_name:
+      description: License name information
+      returned: success
+      type: string
+      sample: Example Inc
+    macaddr:
+      description: The MAC address of the first interface
+      returned: success
+      type: string
+      sample: 52:54:00:4c:e2:07
+    mode:
+      description: Operational mode of the unit
+      returned: success
+      type: string
+      sample: Siparator
+    modules:
+      description: Installed module licenses
+      returned: success
+      type: string
+      sample: failover vpn sip qturn ems qos rsc voipsm
+    patches:
+      description: Installed patches on the unit
+      returned: success
+      type: list
+      sample: []
+    product:
+      description: The product name
+      returned: success
+      type: string
+      sample: Software SIParator/Firewall
+    serial:
+      description: The serial number of the unit
+      returned: success
+      type: string
+      sample: IG-200-839-2008-0
+    systemid:
+      description: The system identifier of the unit
+      returned: success
+      type: string
+      sample: IG-200-839-2008-0
+    unitname:
+      description: The name of the unit
+      returned: success
+      type: string
+      sample: Testname
+    version:
+      description: Firmware version
+      returned: success
+      type: string
+      sample: 6.2.0-beta2
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.ingate.common import (ingate_argument_spec,
+                                                        ingate_create_client)
+
+try:
+    from ingate import ingatesdk
+    HAS_INGATESDK = True
+except ImportError:
+    HAS_INGATESDK = False
+
+
+def make_request(module):
+    # Create client and authenticate.
+    api_client = ingate_create_client(**module.params)
+
+    # Get unit information.
+    response = api_client.unit_information()
+    return response
+
+
+def main():
+    argument_spec = ingate_argument_spec()
+    mutually_exclusive = []
+    required_if = []
```

Comment:
Since the value of `mutually_exclusive` and `required_if` is empty it can be removed and is not required to be passed to `AnsibleModule`